src/composables/use-browser-detection.ts

```diff
@@ -5,7 +5,7 @@ export const useBrowserDetection = () => {
   return app.$ua
 }
 
-export const useBrowserIsMobile = () => {
+export const useBrowserIsMobile = (): boolean => {
```

Comment:
TS should be able to derive the type here, but for some reason WebStorm doesn't pick up the type. Adding this annotation helps